it/src/main/java/com/google/cloud/teleport/it/pubsub/PubsubResourceManagerException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.pubsub;
+
+/** Custom exception for {@link PubsubResourceManager} implementations. */
+public class PubsubResourceManagerException extends RuntimeException {
+
+  public PubsubResourceManagerException(String errorMessage, Throwable err) {
```

Comment:
Maybe make this package private. The class should be public, but I don't think we want any other packages creating one of these exceptions.